torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -77,6 +77,7 @@ py::object PyRRef::localValue() {
     const py::object& value =
         std::dynamic_pointer_cast<OwnerRRef<py::object>>(rref_)->getValue();
 
+    PythonRpcHandler::getInstance().handleException(value);
```

Comment:
UserRRef.toHere() raises the error. Is it because this line?

https://github.com/pytorch/pytorch/blob/63675b1969d5437384851e0e543b2fe9785334ce/torch/csrc/distributed/rpc/rref.cpp#L170

Reply:
no, it is because I added exception in get_value() which could be called in server side:(. 

I updated diff to add exception in localValue() in python/caller side only